CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Unreleased Changes
 ------------------
 
+* Feature - Aws::Record::DirtyTracking - Add the `persisted?`, `new_record?`, and `destroyed` methods to `Aws::Record::DirtyTracking`, which supports use cases where you'd like to see if a record has just been newly initialized, or has been deleted or was a preexisting record retrieved from DynamoDB. Note that these methods are present in `ActiveModel::Model` so you should require that module before `Aws::Record`
```

Comment:
In the changelog, let's avoid `DirtyTracking` and `ItemOperations`. These don't mean anything to an end user, they're basically code organization. Rather, just explain that these methods will now be tacked on to your models.

Also, we may need an "UPGRADING NOTES" bit on ActiveModel, I'll talk with you about that tomorrow.